AEPCore/Tests/EventHubTests/EventHubTests.swift

```diff
@@ -629,7 +633,7 @@ class EventHubTests: XCTestCase {
         eventHub.start()
 
         // test
-        eventHub.createSharedState(extensionName: "mockExtensionTwo", data: SharedStateTestHelper.ONE, event: nil)
+        eventHub.createSharedState(extensionName: "mockExtensionTwo", data: SharedStateTestHelper.ONE.standard, event: nil)
```

Comment:
These tests shouldn't be changed if we overload the API. Let's add new tests for the new APIs.